plugins/modules/system/vdo.py

```diff
@@ -795,34 +699,25 @@ def run_module():
 
             if currentphysblocks > growthresh:
                 result['changed'] = True
-                rc, out, err = module.run_command("%s growPhysical --name=%s"
-                                                  % (vdocmd, desiredvdo))
+                rc, out, err = module.run_command([vdocmd, "growPhysical", "--name=%s" % desiredvdo])
 
         if 'logicalsize' in diffsizeparams.keys():
             result['changed'] = True
-            vdocmdoptions = ("--vdoLogicalSize=" +
-                             diffsizeparams['logicalsize'])
-            rc, out, err = module.run_command("%s growLogical --name=%s %s"
-                                              % (vdocmd,
-                                                 desiredvdo,
-                                                 vdocmdoptions))
+            rc, out, err = module.run_command([vdocmd, "growLogical", "--name=%s" % desiredvdo, "--vdoLogicalSize=%s" % diffsizeparams['logicalsize']])
 
         vdoactivatestatus = processedvdos[desiredvdo]['Activate']
 
-        if ((module.params['activated'] == 'no') and
-                (vdoactivatestatus == 'enabled')):
+        if module.params['activated'] is False and vdoactivatestatus == 'enabled':
```

Comment:
This (and the followings) are bugs that should be mentioned in the changelog fragment: booleans never equaled `no` or `yes`. This must have been broken for a long, long time.

Reply:
done